src/DynamoCore/Configuration/Configurations.cs

```diff
@@ -130,6 +130,16 @@ public class Configurations
         /// Default width of tab
         /// </summary>
         public static readonly int TabDefaultWidth = 250;
+
+        /// <summary>
+        /// Minimum width of tab in the extensions side bar
+        /// </summary>
+        public static readonly int ExtensionsSideBarTabMinWidth = 150;
+
+        /// <summary>
+        /// Border brush for tab items
+        /// </summary>
+        public static readonly string BorderBrushTabItems = "#FF3F4040";
```

Comment:
sorry Ashish, I was not clear, I meant add it to 
https://github.com/DynamoDS/Dynamo/blob/master/src/DynamoCoreWpf/UI/Themes/Modern/DynamoModern.xaml

Reply:
or one of these other xmal files

Reply:
Yup, added it to the https://github.com/DynamoDS/Dynamo/blob/master/src/DynamoCoreWpf/UI/Themes/Modern/DynamoColorsAndBrushes.xaml.